tap_stripe/__init__.py

```diff
@@ -392,6 +393,23 @@ def reduce_foreign_keys(rec, stream_name):
                     rec['lines'][k] = [li.to_dict_recursive() for li in val]
     return rec
 
+def new_list(self, api_key=None, stripe_version=None, stripe_account=None, **params):
+    '''The new list function to overwrite the list() in the ListObject class.'''
+    stripe_object = self._request(  # pylint: disable=protected-access
```

Comment:
This should call through to the original implementation instead of re-implementing it so that it is resilient to SDK version upgrades and gets any features implemented in the underlying library (e.g., security, performance, etc.).